app/views/metadata_fields/_drcf_digital_markets_researches.html.erb

```diff
@@ -0,0 +1,61 @@
+<%= render layout: "shared/form_group", locals: { f: f, field: :digital_market_research_category, label: "Category" } do %>
+  <%= f.select :digital_market_research_category,
+      f.object.facet_options(:digital_market_research_category),
+      {},
+      {
+        class: 'select2 form-control',
+        data:
+        {
+          placeholder: 'Select a category'
+        }
+      }
+  %>
```

Comment:
I have added `include_blank: true`, as single select would break the specs, but maybe I am missing something.

Reply:
Sorry, this was my mistake. I thought it still remained a select multiple because it was a select2. I now notice there isn't the `multiple: true` attribute.

Is the precedence in this app to always use a select2 for any select inputs? It seems a bit heavy duty to me given there's only 2 options here and a non-enhanced select would be more appropriate.

Reply:
Yes it looks like specialist-publisher uses select2 exclusively.

Reply:
Cool - lets keep that then. Sorry for the misunderstanding